src/server/index.js

```diff
@@ -100,15 +100,15 @@ class Jarvis {
       bootJarvis();
     }
 
-    compiler.plugin("watch-run", (c, done) => {
+    compiler.hooks.watchRun.tap("Jarvis", (c, done) => {
```

Comment:
You should leave 'done' and 'done()' (in the next) in webpack5

compiler.hooks.watchRun.tap("Jarvis", (c) => {


<img width="625" alt="image" src="https://user-images.githubusercontent.com/37032760/225035135-529af166-31b1-4415-b6df-9aabee70638c.png">